doc/API-Mastodon.md

```diff
@@ -29,7 +29,91 @@ For supported apps please have a look at the [FAQ](help/FAQ#clients)
 
 ## Entities
 
-These endpoints use the [Mastodon API entities](https://docs.joinmastodon.org/entities/).
+These endpoints use the [Mastodon API entities](https://docs.joinmastodon.org/entities/). With some additional extensions listed below.
+
+### Instance (Version 2) Entities
+Extensions to the [Mastodon Instance::V2 Entities](https://docs.joinmastodon.org/entities/Instance/)
+* `friendica`: Friendica specific properties of the V2 Instance including:
+	* `version`: The Friendica version string
```

Comment:
Indentation is different from the following lines.

Reply:
It is a sub-field of the friendica block.

Reply:
```suggestion
    * `version`: The Friendica version string
```